src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -655,6 +655,10 @@ In this message, the term "bundled" is a short way of saying "included into the
     <comment>The following are literal names and should not be translated: Wear, APK, SignAndroidPackage
 "Wear" is a short version of the full product name "Wear OS" and so should not be translated.</comment>
   </data>
+  <data name="XA4312" xml:space="preserve">
+    <value>Embedding an Android Wear application inside an Android application is deprecated. Distribute the Wear application as a standalone application instead.</value>
```

Comment:
To brainstorm on wording a little, maybe:

```
Referencing an Android Wear application project from an Android application project is deprecated and will no longer be supported in a future version of Xamarin.Android. Remove the Android Wear application project reference from the Android application project and distribute the Wear application as a standalone application instead.
```

Motivation: I did a quick search to see if "embed" or "embedding" appeared on any of the Android Wear docs on docs.microsoft.com. It looks like those terms don't currently appear there, so maybe it's simplest to describe the scenario in generic project referencing terms.

If the project reference terminology feels too imprecise, another option could be to use a phrase like "packaging ... as a resource" to align with the terminology on <https://docs.microsoft.com/xamarin/android/wear/deploy-test/packaging>.